app/Controllers/Api/V1/CommentsController.php

```
@@ -20,60 +22,239 @@
+            'post_id' => 'required|integer|is_natural_no_zero',
+            'content' => 'required|min_length[1]',
+        ];
+
+        $payload = $this->request->getJson(true);
+
+        if (!$payload) {
+            return $this->failValidationErrors('Invalid payload');
+        }
+
+        if (!$this->validateData($payload, $rules)) {
+            return $this->failValidationErrors($this->validator->getError());
```

PHPStan (2.1.54)

[error] Method CodeIgniter\Validation\ValidationInterface::getError() invoked with 0 parameters, 1 required.


(arguments.count)